Zend/zend_closures.c

```diff
@@ -38,6 +38,7 @@ typedef struct _zend_closure {
 	zend_function     func;
 	zval              this_ptr;
 	zend_class_entry *called_scope;
+	HashTable        *static_variables;
```

Comment:
I don't understand way we need this new "static_variables" and can't reuse "func.static_variables" and "static_variables_ptr".

Reply:
Yes, the changes in this file turned out to be unnecessary. I've reverted them.